helm/cluster/templates/clusterapi/cluster.yaml

```diff
@@ -2,35 +2,35 @@ apiVersion: cluster.x-k8s.io/v1beta1
 kind: Cluster
 metadata:
   annotations:
-    {{- with .Values.metadata.description }}
+    {{- with $.Values.global.metadata.description }}
```

Comment:
Updating Values usage here as this PR moves top-level properties into `Values.global`.